### ISSUE TITLE
A question about training.

### ISSUE THREAD

**Issue by goodstudent9** [zd11024/NaviLLM#19]
I want to make sure I am right.
I have 2 A100, so if I want to reproduce everything like yours, I only need to set the --gradient_accumulation_step to 32, right?
I don't need to change num_steps_per_epoch, num_epochs and others, right?

Thank you for your reply.

**Comment by zd11024** [comment]
The actual training step is equal to `num_steps_per_epoch * num_epochs / gradient_accumulation_step `. If you set `gradient_accumulation_step` to 32, you need to scale either `num_steps_per_epoch` or `num_epochs` by 4 times.

**Comment by goodstudent9** [comment]
Thank you so much for your reply! I understand it ! 😊

But I found your log is different from your paper. **Especially with CVDN's SPL on Val-Unseen dataset.**🙋
I think the chart in your paper is with-pretrain, which means 40 epochs pretrain+20 epoch's finetune. But that seems like there is abig difference between log file and chart. Is that normal? How much difference should be ok usually?
![image](https://github.com/user-attachments/assets/c9e2ed44-054e-4f21-ba24-cf19db2cf53e)
![image](https://github.com/user-attachments/assets/37d81b25-3c9f-4cb1-a1f1-bde9680d550a)

**Comment by zd11024** [comment]
The results on the CVDN Val set results are 6.16 and 5.91 in the chart and log, respectively, which is within a reasonable range.

**Comment by goodstudent9** [comment]
> The results on the CVDN Val set results are 6.16 and 5.91 in the chart and log, respectively, which is within a reasonable range.

**Oh, thank you so much for your immediate reply.**  🙏☕️
I just reference **SPL in CVDN log** according to your paper "We report SPL for CVDN......." which is about 14.11. 
But that seems like you are talking about **dist_to_end_reduction** ?

**Comment by zd11024** [comment]
Yes, sorry for this mistake. Actually, for CVDN, Goal Progress (`dist_to_end_reduction`) is adopted as the primary metric.

**Comment by goodstudent9** [comment]
I  understand!
So in your paper's chart, you are actually talking about `dist_to_end_reduction`, right?

**Comment by zd11024** [comment]
yes
